superset-frontend/src/components/MetadataBar/MetadataBar.tsx

```diff
@@ -100,7 +100,6 @@ const StyledItem = styled.div<{
     }
     & .metadata-text {
       min-width: ${TEXT_MIN_WIDTH}px;
-      overflow: hidden;
```

Comment:
Are you sure that removing overflow is needed here? I wonder if it won't break anything

Reply:
looking at the metabar in the ui it didn't effect the styling from what i could see

Reply:
It's required if the text in metadata bar is too long. 1 screenshot is with `overflow: hidden`, second is without

<img width="526" alt="image" src="https://github.com/apache/superset/assets/15073128/c836f51f-633a-4e55-ae82-2dff1e0f4544">

<img width="637" alt="image" src="https://github.com/apache/superset/assets/15073128/38b73304-c111-421e-9a3c-f5eaad59470d">